src/pages/index.js

```diff
@@ -78,34 +111,103 @@ picturePopup.setEventListeners();
 const userInfo = new UserInfo({
   nameSelector: "#profile-title",
   jobSelector: "#profile-description",
+  avatarSelector: "#profile-image",
+});
+
+api.getCurrentUser().then((result) => {
+  userInfo.setUserInfo(result.name, result.about);
+  userInfo.setAvatar(result.avatar);
 });
 
 /******************
  * EVENT HANDLERS *
  ******************/
 function handleProfileEditSubmit(e, { title, description }) {
   e.preventDefault();
-  userInfo.setUserInfo(title, description);
-  profilePopup.close();
+  profilePopup.loading(true);
+  api
+    .updateUser(title, description)
+    .then((res) => res.json())
+    .then((result) => {
+      console.log(result);
+    })
+    .catch((err) => console.log(err))
+    .finally(() => {
+      userInfo.setUserInfo(title, description);
```

Comment:
`      userInfo.setUserInfo(title, description);
`

You need to update `DOM` only in blocks `then` after a successful response from the server so that when there is an error it wouldn’t change


**Please, correct it everywhere**